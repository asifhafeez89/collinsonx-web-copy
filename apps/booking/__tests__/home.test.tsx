import * as React from 'react';
import renderer from 'react-test-renderer';
import Home from '../pages/index';

describe('<Home />', () => {
  it('renders outline variant', () => {
    const tree = renderer
      .create(
        <Home />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
