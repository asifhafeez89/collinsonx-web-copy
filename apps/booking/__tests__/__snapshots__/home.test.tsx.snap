// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders outline variant 1`] = `
<p>
  Hello World
</p>
`;
